### ISSUE TITLE
Underscores not preserved when building query function

### ISSUE THREAD

**Issue by evan-sheehan** [ticruz38/graphql-codegen-svelte-apollo#33]
Not sure if this is intentional or just not  caught up with the latest version of other packages, but I noticed graphql-codegen-svelte-apollo builds the query/subscription function by converting the needed type names to pascal case. This is in conflict with whichever package generates those types (I think it's typescript-operations) where the underscores are preserved by default. For example, I might have a query I name Query_Users. When I run graphql-codegen with typescript-operations and graphql-codegen-svelte-apollo, I will get a type named Query_UsersQuery. Graphql-codegen-svelte-apollo will then  generate a function with the proper name, "Query_Users", but will reference the query type generated earlier as "QueryUsersQuery"  instead of the original name with the underscore.

Anyways, super long winded explanation of an easy to explain issue. The point is I don't  believe this library is up to date with the standard naming convention settings of graphql-codegen and/or other libraries under its wing.

**Comment by ticruz38** [comment]
Fair enough, for now a quick fix would be not to use underscore for you, this is an edge case I did not think of. 
Would need to check what library is used by typescript-operations to transform query names
